public/locales/ar/translation.json

```diff
@@ -911,7 +911,7 @@
         "noResults": "لم تسفر التصنيفات الحالية عن أية نتائج."
     },
     "wineExplanation": {
-        "proton-ge": "Proton-GE-Proton هو نوع Proton تم إنشاؤه بواسطة Glorious Eggroll. من المفترض استخدامه في Steam، لكن بعض الألعاب خارج Steam قد تعمل بشكل أفضل مع هذا المتغير. ويوفر سجلات غير مجدية في الغالب لاستكشاف الأخطاء وإصلاحها.",
+        "proton-ge": "GE-Proton هو نوع Proton تم إنشاؤه بواسطة Glorious Eggroll. من المفترض استخدامه في Steam، لكن بعض الألعاب خارج Steam قد تعمل بشكل أفضل مع هذا المتغير. ويوفر سجلات غير مجدية في الغالب لاستكشاف الأخطاء وإصلاحها.",
```

Comment:
hmmm I wonder how these changes work with weblate

will this update weblate? or weblate will have the old stuff and the next PR it will override these changes again?

Reply:
I'm pretty sure that Weblate will update with the changes from here. The relation of repo<->Weblate goes both ways (at least it does for the English translations, and I don't see why other languages should be different)